### PR TITLE
Add missing docs

### PR DESCRIPTION
**Pull request by abarth**

### CODE REVIEW

File: packages/flutter/lib/src/services/text_formatter.dart [hunk 1]
```diff
@@ -4,38 +4,38 @@
...
 /// to provide as-you-type validation and formatting of the text being edited.
-/// 
+///
 /// Text modification should only be applied when text is being committed by the
-/// IME and not on text under composition (i.e. when 
-/// [TextEditingValue.composing] is collapsed). 
-/// 
-/// Concrete implementations [BlacklistingTextInputFormatter], which removes 
-/// blacklisted characters upon edit commit, and 
+/// IME and not on text under composition (i.e. when
```
Comment (Hixie) [inline packages/flutter/lib/src/services/text_formatter.dart]: `i.e. only when`... to make it clear the i.e. applies to the first clause and not the second.

File: packages/flutter/lib/src/services/text_formatter.dart [hunk 2]
```diff
@@ -48,50 +48,51 @@ abstract class TextInputFormatter {
...
 }
 
-/// A [TextInputFormatter] that prevents the insertion of blacklisted 
+/// A [TextInputFormatter] that prevents the insertion of blacklisted
 /// characters patterns.
-/// 
-/// Instances of blacklisted characters found in the new [TextEditingValue]s 
+///
+/// Instances of blacklisted characters found in the new [TextEditingValue]s
 /// will be replaced with the [replacementString] which defaults to ``.
```
Comment (Hixie) [inline packages/flutter/lib/src/services/text_formatter.dart]: to the empty string

File: packages/flutter/lib/src/services/text_formatter.dart [hunk 3]
```diff
@@ -48,50 +48,51 @@ abstract class TextInputFormatter {
...
+///
 /// Since this formatter only removes characters from the text, it attempts to
 /// preserve the existing [TextEditingValue.selection] to values it would now
 /// fall at with the removed characters.
-/// 
+///
 /// See also:
-///  
+///
 ///  * [TextInputFormatter].
```
Comment (Hixie) [inline packages/flutter/lib/src/services/text_formatter.dart]: which...
which...

File: packages/flutter/lib/src/services/text_formatter.dart [hunk 4]
```diff
@@ -117,23 +118,26 @@ class BlacklistingTextInputFormatter extends TextInputFormatter {
...
 ///  * [BlacklistingTextInputFormatter].
```
Comment (Hixie) [inline packages/flutter/lib/src/services/text_formatter.dart]: which which

**Comment by Hixie** [comment]
LGTM

**Comment by xster** [comment]
LGTM

**Comment by xster** [comment]
Turned trim trailing whitespace on my editor now :)

**Comment by abarth** [comment]
> Turned trim trailing whitespace on my editor now :)

Sorry about that.  I didn't notice that until after I posted this CL.

**Pull request merged**